src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -159,10 +172,7 @@ public function getDefinition()
 $c = $d    xor    $e;
 $f    -=  1;
 ',
-                    [
-                        '=' => 'align',
-                        'xor' => null,
-                    ]
+                    ['operators' => ['=' => 'align', 'xor' => null]]
```

Comment:
no constant ?

Reply:
you mean for `operators` or for all options, is it worth it?

Reply:
sorry, for `align`